### ISSUE TITLE
Mounting partitions not working properly

### ISSUE THREAD

**Issue by cryptblood1986** [GDX-X/PFS-BatchKit-Manager#34]
It opens Quick access in file explorer instead.

**Comment by GDX-X** [comment]
Yes and your partition must be in the explorer if it does not appear verified that you have not made a typo or reboot your comptuer or uninstall / reinstall the driver

**Comment by almircanella** [comment]
To me does not work either (win11 ver22h2 22621.1265). Driver install just fine, but when mounted, just open quick access. Nothing was mounted. I've already unninstalled/installed again, but no dice.

**Comment by lichtmetzger** [comment]
Works for me on the latest Win11 build. Rebooted to enable testsigning, installed the driver, can mount partitions.

I don't use SecureBoot, though, if that's what makes the difference.
